### PR TITLE
Newsletter march

### PR DESCRIPTION
**Pull request by nikishe**

### CODE REVIEW

**Comment by jkwmoore** [comment]
I've made some corrections, please review and let me know if you're happy (or not as it is perfectly plausible I introduce some mistakes.)

**Review by jkwmoore — APPROVED** [review]
Approved by me if everyone is happy.

**Review by willfurnass — APPROVED** [review]
Nice and punchy!

File: newsletter/20240301.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.. _nl20240301:
+
+News - 2024-03-01
+=================
+
+Hello everyone! We have made it to spring, at least for those who count seasons meteorologically. For those who count seasons astronomically, stay strong; there's just 20 more days of winter.
+
+- This month we are are excited to invite you all to our **Research IT Forum**. More details are in the :ref:`Upcoming Events <upcoming_events>` section.
```
Comment (Carldkennedy) [inline newsletter/20240301.rst:8]:
```suggestion
- This month we are are excited to invite you all to our **Research IT Forum**, details are in the :ref:`Upcoming Events <upcoming_events>` section. 
```

File: newsletter/20240301.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ 
+- Python Programming 2 - 05/03/2024
+- Python Programming 3 - 12/03/2024
+- High Performance Computing - 05/03/2024
+- Basics of Supervised Machine Learning - 08/03/2024
+- Introduction to Matlab 1 - 14/03/2024
+- Introduction to Matlab 2 - 21/03/2024
+- Introduction to R - 26/03/2024
+- Deep learning - 04/04/2024 to 05/04/2024 (2 day course)
+
```
Comment (Carldkennedy) [inline newsletter/20240301.rst:55]: Wolfram's free upcoming training https://events.wolfram.com/

**Review by Carldkennedy — COMMENTED** [review]
LGTM added a few comments/suggestions

File: newsletter/20240301.rst (hunk 1 above)
Comment (jkwmoore) [reply newsletter/20240301.rst:8]: Now you mention it, that's a double "are".

File: newsletter/20240301.rst (hunk 2 above)
Comment (jkwmoore) [reply newsletter/20240301.rst:55]: Did we have some we specifically wanted to mention?

File: newsletter/20240301.rst (hunk 1 above)
Comment (jkwmoore) [reply newsletter/20240301.rst]: We still have a double are here.

Comment (jkwmoore) [reply newsletter/20240301.rst]: Resolving here to add a comment in the main PR sequence.

File: newsletter/20240301.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.. _nl20240301:
+
+News - 2024-03-01
+=================
+
+Hello everyone! We have made it to spring, at least for those who count seasons meteorologically. For those who count seasons astronomically, stay strong; there's just 20 more days of winter.
+
+- This month we are are excited to invite you all to our **Research IT Forum**, details are in the :ref:`Upcoming Events <upcoming_events>` section.
```
Comment (jkwmoore) [inline newsletter/20240301.rst:8]:
```suggestion
- This month we are excited to invite you all to our **Research IT Forum**, details are in the :ref:`Upcoming Events <upcoming_events>` section. 
```

FYI a grammar check still suggests this should be two separate sentences.

**Review by jkwmoore — APPROVED** [review]
This now looks LGTM.

For future note,  within a PR can we please minimize the number of commits keeping as many contiguous changes (in a sensible way) to a single commit and provide an appropriately descriptive message for each.

When merging this PR, please ensure a squash commit is used to reduce the amount of commits sensibly.

**Pull request merged**